cli/src/main/java/de/jplag/cli/CLI.java

```diff
@@ -1,290 +1,120 @@
 package de.jplag.cli;
 
-import static picocli.CommandLine.Model.UsageMessageSpec.SECTION_KEY_DESCRIPTION_HEADING;
-import static picocli.CommandLine.Model.UsageMessageSpec.SECTION_KEY_OPTION_LIST;
-import static picocli.CommandLine.Model.UsageMessageSpec.SECTION_KEY_SYNOPSIS;
-
-import java.awt.Desktop;
 import java.io.File;
 import java.io.FileNotFoundException;
 import java.io.IOException;
-import java.net.URI;
-import java.security.SecureRandom;
-import java.util.Arrays;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Random;
-import java.util.Set;
-import java.util.stream.Collectors;
 
 import org.slf4j.ILoggerFactory;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import de.jplag.JPlag;
 import de.jplag.JPlagResult;
-import de.jplag.Language;
 import de.jplag.cli.logger.CollectedLoggerFactory;
 import de.jplag.cli.logger.TongfeiProgressBarProvider;
-import de.jplag.cli.server.ReportViewer;
-import de.jplag.clustering.ClusteringOptions;
-import de.jplag.clustering.Preprocessing;
+import de.jplag.cli.picocli.CliInputHandler;
 import de.jplag.exceptions.ExitException;
 import de.jplag.logging.ProgressBarLogger;
-import de.jplag.merging.MergingOptions;
 import de.jplag.options.JPlagOptions;
-import de.jplag.options.LanguageOption;
-import de.jplag.options.LanguageOptions;
-import de.jplag.reporting.reportobject.ReportObjectFactory;
-
-import picocli.CommandLine;
-import picocli.CommandLine.Model.CommandSpec;
-import picocli.CommandLine.Model.OptionSpec;
-import picocli.CommandLine.ParseResult;
 
 /**
  * Command line interface class, allows using via command line.
  * @see CLI#main(String[])
  */
 public final class CLI {
```

Comment:
In all other classes you use `Cli` but here it is capitalized

Reply:
No, `CliFooBar` is the standard way to work with established abbreviations in Java identifiers.